CONTRIBUTING.md

```diff
@@ -64,7 +64,7 @@ Check out [this video for a live walkthrough](https://youtu.be/Tx257WpNsxc) or f
 - Run `cd ./tmp` in Terminal 2
 - Run `npx create-nx-workspace@16.0.0` in Terminal 2
 
-If you have problems publishing, make sure you use Node 16 and NPM 6 or 8.
+If you have problems publishing, make sure you use Node 18 and NPM 6 or 8.
```

Comment:
Should we say something about the npm version?

Reply:
I would put whatever version is coming with node 18.

Reply:
Ok, I put `npm v8`